crocket/src/main/java/com/crocket/Model.java

```diff
@@ -152,16 +166,53 @@ public void updateAim(int degrees) {
         directionLine.setDegreeAngle(degrees);
     }
 
+    public void aimRight() {
+        directionLine.incrementDegreeAngle();
+    }
+
+    public void aimLeft() {
+        directionLine.decrementDegreeAngle();
+    }
+
     public void setPlayers(List<Player> players) {
         this.players = players;
+
+        activePlayer = players.get(0);
+
+        for (Player player : players) {
+            entities.add(player.getBall());
```

Comment:
This method closely resembles addPlayer. Can we consider incorporating it into this method?

Reply:
Fixed